packages/hardhat-ovm/src/index.ts

```diff
@@ -35,6 +35,7 @@ const OVM_POLLING_INTERVAL = 50
 /**
  * Find or generate an OVM soljson.js compiler file and return the path of this file.
  * We pass the path to this file into hardhat.
+ *
```

Comment:
Just curious, is this a linter rule? I feel like there'd be more changes like this if so. Either way I'm fine with it, white space is nice.

Reply:
I think it is the `jsdoc/newline-after-description` rule. Right now it's a bit inconsistent across packages (I added the "off" version to make the linter happy in some places but I can make it consistent like the above.

Reply:
IMO, we can/should deal with the pain of a larger diff that will come from being more strict now.
So, I think we should activate any rules that are:
1. consistent with current style
2. don't result in an _excessively_ burdensome diff

I know 2 is vague, but trust your judgement. :)